src/System.Windows.Forms/tests/IntegrationTests/System.Windows.Forms.IntegrationTests.Common/TestDataSources.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace System.Windows.Forms.IntegrationTests.Common
+{
+    public class TestDataSources
+    {
+        public static List<Person> s_persons = new List<Person>()
```

Comment:
I'd rather not share mutable data unless it is absolutely necessary. We're already having enough problems with our tests as is. 
Is it possible to change this into the following?
```suggestion
        public static List<Person> GetPersons() => new List<Person>()
```

Reply:
Thanks!